### PR TITLE
flip password feedback ui logic and use clearer flag name

### PR DESCRIPTION
**Pull request by czue**
http://manage.dimagi.com/default.asp?222974

@millerdev / anyone. cc @calellowitz

### CODE REVIEW

File: corehq/apps/registration/views.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def register_user(request):
 
         context.update({
             'form': form,
-            'legacy_password': getattr(settings, 'ENABLE_DRACONIAN_SECURITY_FEATURES', False),
+            'show_password_feedback': not getattr(settings, 'ENABLE_DRACONIAN_SECURITY_FEATURES', False),
```
Comment (snopoke) [inline corehq/apps/registration/views.py]: Are you sure that's correct? I think you want to show the feedback if `ENABLE_DRACONIAN_SECURITY_FEATURES` is True.

Comment (czue) [inline corehq/apps/registration/views.py]: i'm pretty sure it's correct. if `ENABLE_DRACONIAN_SECURITY_FEATURES` then we have stricter password requirements which are different from the front end library. context: https://github.com/dimagi/commcare-hq/pull/10864#issuecomment-199340457

File: corehq/apps/registration/templates/registration/create_new_user.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
     {% url "eula_basic" as eula_url %}
     {% include 'hqstyle/partials/load_into_modal.html' with modal_id="eulaModal" load_from=eula_url %}
     <script src="{% static 'zxcvbn/dist/zxcvbn.js' %}"></script>
-    {% if legacy_password %}
+    {% if show_password_feedback %}
```
Comment (millerdev) [inline corehq/apps/registration/templates/registration/create_new_user.html]: d'oh! classic logic reversal

File: corehq/apps/registration/views.py (hunk 1 above)
Comment (snopoke) [inline corehq/apps/registration/views.py]: Clarified offline, logic is fine but I would also exclude the `zxcvbn` lib along with `password.js`. There are also 3 other places where this frontend validation is done which should receive the same treatment:
* password reset
* change password
* accept invite

Comment (czue) [inline corehq/apps/registration/views.py]: whoops, makes sense. made an interrupt ticket here: http://manage.dimagi.com/default.asp?223037

Comment (millerdev) [inline corehq/apps/registration/views.py]: yeah, just noticed the same thing. Will fix

**Pull request merged**